### DISCUSSION TITLE
Logout all after password change

### DISCUSSION THREAD

**Discussion by elsacutealways** [pocketbase/pocketbase#2336] [Q&A]
How to logout all sessions after user password change?

**Comment by ganigeorgiev** [comment]
PocketBase is completely stateless and there are no sessions in the traditional sense. A request is considered authenticated if it has a valid `Authorization: TOKEN` header sent with it.

When a user or admin change their password, all previously issued tokens are invalidated, aka. they are automatically "signed out".
